web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/StoreInfo/page/UpdateStoreInfoForm/index.tsx

```diff
@@ -110,6 +115,14 @@ export const UpdateStoreInfoForm = (props: UpdateStoreInfoFormProps) => {
     },
   });
 
+  useEffect(() => {
```

Comment:
Why do we need reset form on every dependencies change? 
Leave comment here too, please

Reply:
default values are not reactive so we use the hook in order to make it update when the view mode changes to verified

Reply:
But I updated to be specifically for those values